apps/src/templates/projects/StartNewProject.jsx

```diff
@@ -155,3 +156,30 @@ const styles = {
     width: '100%',
   },
 };
+
+/**
+ * This is a version of the new project selection that is overridable by a region
+ * configuration.
+ *
+ * This is done via a configuration in, for instance, /config/global_editions/fa.yml
+ * via a paths rule such as:
+ *
+ * ```
+ * pages:
+ *   # All pages
+ *   - path: /
+ *     components:
+ *       LtiFeedbackBanner: false
+ *       StartNewProject:
+ *         canViewFullList: false
+ * ```
+ */
+const RegionalStartNewProject = props => (
+  <GlobalEditionWrapper
+    component={StartNewProject}
+    componentId="StartNewProject"
+    props={props}
```

Comment:
Allow a regional configuration... for us, it will be to prevent showing a full listing of potential projects. We want to curate the possible projects within a region to those with good translations relevant in that region.